eng/native/configurecompiler.cmake

```
@@ -696,6 +696,11 @@ if (CLR_CMAKE_HOST_UNIX)
 endif(CLR_CMAKE_HOST_UNIX)
 
 if(CLR_CMAKE_TARGET_UNIX)
+  if(CLR_CMAKE_HOST_ARCH_ARM64)
```

```suggestion
  if(CLR_CMAKE_TARGET_ARCH_ARM64 AND CLR_CMAKE_TARGET_LINUX)
```
nit: perhaps it is more suitable in the previous block? (with rest of the `add_compile_options`)

---

Done!